common.mk

```diff
@@ -71,6 +71,7 @@ build-image:
 	@# Add --pull so that we are using the latest base image.
 	@# The build context is the parent directory
 	docker build --pull \
+		--provenance=false \
```

Comment:
Why?

Reply:
This is related to the unknown manifest mentioned yesterday. I found out that they are [Provenance attestations](https://docs.docker.com/build/metadata/attestations/slsa-provenance/). So I disabled it in build.